translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+      </trans-unit>
+      <trans-unit id="4864789526486078372" datatype="html">
+        <source>Every piece has three neighboring triangular spaces (2 on the sides).
+        When all neighboring spaces except one are occupied, and one opponent moves to that last free space, your piece is captured!
+        However, it is possible to place a piece between 3 of the opponent&apos;s pieces (or 2 on the side) without being captured.&lt;br/&gt;&lt;br/&gt;
+        You&apos;re playing Light, make a capture.</source>
+        <target>Chaque pièce a trois cases triangulaires voisines (2 sur les bords).
+        Quand toutes les cases voisines sauf une sont occupées, et qu&apos;une pièce de l'adversaire vient se déplacer sur cette dernière case libre, votre pièce est capturée !
+        Cependant, il est possible pour un joueur de se placer entre 3 pièces adverses (ou 2 contre un bord) sans être capturé.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez Clair, effectuez une capture</target>
+      </trans-unit>
+      <trans-unit id="1766583918856668821" datatype="html">
+        <source>Failed, you have not captured any piece.</source>
+        <target>Raté, vous n&apos;avez pas capturé de pièce !</target>
+      </trans-unit>
+      <trans-unit id="8225905705628695723" datatype="html">
+        <source>Gain a tile</source>
+        <target>Gagner une tuile</target>
+      </trans-unit>
+      <trans-unit id="362405963474133663" datatype="html">
+        <source>When a tile is empty, it can become removable from the board.
+        For it to be removable, at least three of its sides must be free, and these should be consecutive sides.
+        Note that if one empty tile, neighboring a tile that was just removed, also becomes removable, it will be removed too.
+        For example, here, the topmost dark piece will not disconnect its tile when leaving it.
+        But, by leaving the bottom left tile, two tiles will be removed.&lt;br/&gt;&lt;br/&gt;
+        Perform a move that lets you gain two tiles.</source>
+        <target>Quand une tuile est quittée, elle devient potentiellement enlevable du plateau.
+        Pour qu&apos;elle soit enlevée, il faut qu'au moins trois de ses bords soient libres, et qu&apos;ils soient l&apos;un à côté de l&apos;autre.
+        Notez que si une tuile vide, voisine d&apos;une tuile qu&apos;on vient de retirer, devient retirable, elle sera retirée.
+        Par exemple, ci-dessous, en quittant sa tuile le pion foncé le plus haut ne déconnectera pas celle-ci !
+        Mais en quittant la tuile en bas à gauche, deux tuiles seront enlevées.&lt;br/&gt;&lt;br/&gt;
+        Effectuez un mouvement pour récupérer deux tuiles.</target>
+      </trans-unit>
+      <trans-unit id="4051956669507633461" datatype="html">
+        <source>Failed, you have not gained the two tiles that you could!</source>
+        <target>Raté, vous n'avez pas récupérer les deux tuiles que vous pouviez !</target>
+      </trans-unit>
```

Comment:
J'ai l'impression qu'il manque une fil de phrase dans les deux langues.

Reply:
ajouté un "try again!"

Reply:
Mmh, je pensais plutôt une continuation genre "que vous pouviez retirer"  et "that you could have gained" :D

Reply:
hmm ça fait pas doublon ? Ou plutôt "les deux tuiles alors que vous le pouviez" / "although you could"

Reply:
oui ça marche aussi !